### ISSUE TITLE
Try finding a nicer way to write SQL

### ISSUE THREAD

**Issue by lawrencejones** [lawrencejones/pg2sink#32]
Perhaps using https://github.com/go-jet/jet to generate a library for the database schema? Something to make the program compile-safe while reducing boilerplate.

**Comment by lawrencejones** [comment]
```
jet -source=PostgreSQL -host=localhost -port=5432 -user=pg2sink -dbname=pg2sink -schema=pg_catalog -path=./tmp
```

**Comment by lawrencejones** [comment]
Solved by #36
